cranelift/fuzzgen/src/function_generator.rs

```diff
@@ -454,6 +455,36 @@ where
         Ok(())
     }
 
+    fn generate_switch(&mut self, builder: &mut FunctionBuilder) -> Result<()> {
+        let _type = *self.u.choose(
+            &[
+                I8, I16, I32, I64, // TODO: I128
```

Comment:
For i128, maybe you could generate two variables and then use `iconcat` if `get_variable_of_type` doesn't support it directly?

Reply:
I'm doing exactly that [here](https://github.com/afonso360/wasmtime/commit/b5802659e58b6eccb6b469483ce4301c11fe6e8b#diff-226e0f113ddbbcedaa5265109a951e52d30aa35307537684f56a5e6bedc9ba0eR323-R328)!

But we need a few additional bits to introduce I128 support.

Right now the fuzzer is hitting #4498 and #4406 when I add I128 support. I can add support for I128 values and work around those two issues by disabling those ops for i128.

I think it would be best to do this in a separate PR.

Reply:
You shouldn't hit https://github.com/bytecodealliance/wasmtime/issues/4498 for the Switch api, right? Only when using `br_table` directly.

Reply:
I don't think so, but we will hit the special cased lowering. I'll rebase the i128 patch after merging this, and disable the crashing ops so that we can start fuzzing Switch's i128.